src/types.ts

```diff
@@ -27,3 +27,5 @@ type Breakpoints = "xs" | "sm" | "md" | "lg" | "xl";
 export type ResponsiveProp<Key extends string, T = number | string> = {
   [key in `${Key}-${Breakpoints}` | Key]?: T;
 };
+
+export type SizeValue = `${number}px` | `${number}`;
```

Comment:
Could you point me to the TypeScript docs for `${number}`?
This seems a bit relaxed to me since it would allow numbers like `-45.928`.

Reply:
unfortunately this is the best we can get for now. There are a few proposals to expand it to allow only positive integers / regex type checking strings etc but the point of this type was to add some more safety than just plain string.

My expectation would be that these values are checked against and an error is thrown in runtime if the values do not match what is expected.